File: SofiForce.Models/Models/SearchModels/RouteSetupSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class RouteSetupSearchModel : BaseSearchModel
    {

		public Int32? BranchId { get; set; }
		public Int32? RouteTypeId { get; set; }
		public Int32? IsActive { get; set; }
		
	}
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/RouteSetupSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface RouteSetupSearchModel extends BaseSearchModel {
    branchId: number | null;
    routeTypeId: number | null;
    branchCode: string | null;
    routeTypeCode: string | null;
    isActive: number | null;
}